Doc/library/io.rst

```diff
@@ -728,6 +728,15 @@ than raw I/O does.
 
       Return :class:`bytes` containing the entire contents of the buffer.
 
+   .. method:: peek(size=1, /)
+
+      Return bytes from the current position onwards without advancing the position.
+      At least one byte of data is returned if not at EOF.
+      Return an empty :class:`bytes` object at EOF.
+      If the size argument is less than one or larger than the number of available bytes,
+      a copy of the buffer from the current position until the end is returned.
```

Comment:
I can understand that size=-1 or size=None return the whole content. But I'm surprised that size=0 returns something different than an empty string or raise an exception.

I suggest to return an empty string when peek(0) is called, it would be similar to read(0).

Reply:
Agreed, changed now.

This was originally for (perceived) consistency with `BufferedReader.peek()`, which does not return empty bytes objects for size=0. But then `BufferedReader.peek()` ignores the size entirely anyway.